src/core_plugins/tile_map/public/base_maps_visualization.js

```diff
@@ -134,13 +134,8 @@ export function BaseMapsVisualizationProvider(serviceSettings) {
       if (!this._baseLayerConfigured()) {
         try {
           const tmsServices = await serviceSettings.getTMSServices();
-          const firstRoadMapLayer = tmsServices.find((s) => {
-            return s.id === 'road_map';//first road map layer
-          });
-          const fallback = firstRoadMapLayer ? firstRoadMapLayer : tmsServices[0];
-          if (fallback) {
-            this._setTmsLayer(firstRoadMapLayer);
-          }
+          const baseLayer =  tmsServices[0]; // Gets layer from config (if provided), otherwise defaults to EMS
```

Comment:
there's no guarantee there will be a first layer in this list. this is because people can turn off connecting to EMS with `includeElasticMapService`.

We should double check too if this actually fixes the UX. The UX should not show the layer from the .yml, if the road map is selected.